scripts/generate_test_matrix.py

```diff
@@ -93,6 +98,33 @@
     "deep": ["exprstmt"],
 }
 
+ALPHA_FEATURES = {
```

Comment:
I don't think this is set in stone yet. I would hold off here until we've finalized the language maturity doc :+1:

Reply:
ah yes, good catch